src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -1308,7 +1315,7 @@ namespace FanCoilUnits {
             }
         }
         // Set the inlet node mass flow rate
-        if ((GetCurrentScheduleValue(FanCoil(FanCoilNum).SchedPtr) > 0.0 || DataHVACGlobals::ZoneCompTurnFansOn) &&
+        if (((GetCurrentScheduleValue(FanCoil(FanCoilNum).SchedPtr) > 0.0 && GetCurrentScheduleValue(FanCoil(FanCoilNum).fanAvailSchIndex) > 0.0) || DataHVACGlobals::ZoneCompTurnFansOn) &&
```

Comment:
This line seems long, was this formatted by Clang?

Reply:
I did not try to change the format, it must be auto-formated by Clang.

Reply:
If you select/highlight this line and then on the Edit tab choose Advanced and select "Format Selection" clang should format the line. Do the same for line 3548.